gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -1047,6 +1004,11 @@ gst_tensor_filter_transform_caps (GstBaseTransform * trans,
   silent_debug_caps (caps, "from");
   silent_debug_caps (filter, "filter");
 
+  if (direction == GST_PAD_SINK)
+    pad = GST_BASE_TRANSFORM_SRC_PAD (trans);
```

Comment:
If ```direction == GST_PAD_SINK```, then ```pad = GST_BASE_TRANSFORM_SRC_PAD (trans)```.
Is it right?

Reply:
yes, direction to sink then return src pad caps.